feasibility-triangle/troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,285 @@
+# Troubleshooting Feasibility Triangle
+
+The feasibility triangle can be composed of the following components.
+To debug your triangle it is important that you check each component, starting with the last component in the queue
+and working your way towards the Middleware, which connects to the central portal.
+
+1. AKTIN - FLARE (FHIR Search) - FHIR Server (not CQL ready)
+2. AKTIN - FHIR Server (CQL ready)
+3. DSF - FLARE (FHIR Search) - FHIR Server (not CQL ready)
+4. DSF - FHIR Server (CQL ready)
+
+=> Debugging Route
+1. FHIR Server -> FLARE check translate ->  FLARE check execute -> AKTIN
+2. FHIR Server -> CQL check execution -> AKTIN
+3. FHIR Server -> FLARE check translate ->  FLARE check execute -> DSF
+4. FHIR Server -> CQL check execution -> DSF
+
+
+Additionally you can use the FLARE tool to check if your data is loaded into the FHIR server correctly.
+See DQA below.
+
+For our monitoring we currently use the queries specified here <https://github.com/medizininformatik-initiative/feasibility-monitoring>
+in the `input-queries.json` file. These will be updated regularly to reflect the newest tests and implemented FHIR modules.
+
+> **Before running any tests update the test queries by executing `bash update-test-queries.sh`**
+
+
+## Communicating problems
+
+If you encounter a problem with any of the components and you have identified in which component the error occurs please create an issue for the respective component directly:
+Blaze: <https://github.com/samply/blaze>
+FLARE: <https://github.com/medizininformatik-initiative/flare>
+AKTIN: <https://github.com/medizininformatik-initiative/feasibility-aktin-plugin>
+DSF: <https://github.com/medizininformatik-initiative/feasibility-dsf-process> 
+
+If you cannot place the error directly use this repository to create an issue:
+<https://github.com/medizininformatik-initiative/feasibility-deploy>
+
+## FHIR Server
+
+Check if your FHIR server is running. In the default installation execute `curl http://localhost:8081/fhir/Patient?_summary=count`
+This should return a result as follows:
+
+```
+{"id":"DCCM7GJX6LIW2SIL","type":"searchset","total":12040,"link":[{"relation":"self","url":"http://localhost:8081/fhir/Patient?_summary=count&_count=50&__t=12041"}],"resourceType":"Bundle"}
+```
+
+If this does not return a result check the logs of your fhir server, for default setup `docker logs -f feasibility-deploy_fhir-server_1`
+and contact our team or create an issue here: https://github.com/samply/blaze
+
+Note: If you have a FHIR server other than Blaze please contact the appropriate vendor or support team.
+
+
+## FLARE
+
+To see if flare is running use `docker ps` to list all your containers currently running.
```

Comment:
Wouldn't is be better to teach people use docker-compose ps and logs instead of plain docker? Because with docker-compose the service names are clear, so we have no need the discover the container names first.